package.json

```diff
@@ -11,7 +11,8 @@
     "lint": "eslint index.js */*.jsx",
     "lint-staged": "lint-staged",
     "prettier": "prettier -l *.js */*.jsx sample/webpack.config.js",
-    "prettier:write": "prettier --write *.js */*.jsx sample/webpack.config.js"
+    "prettier:write": "prettier --write *.js */*.jsx sample/webpack.config.js",
+    "sample": "$(npm bin)/http-server ./sample"
```

Comment:
`$(npm bin)` doesn't work on all shells or Windows, shouldn't `http-server ./sample` work without it?

Reply:
Nah, can't just execute a local npm package like that.

Reply:
Wait, why not? Works perfectly for `prettier`, `lint-staged` and `eslint` in this very same file. Also, just tested specifically for `http-server` and it works without any issues.